### PR TITLE
Put Admin un/pw into ENV vars

### PR DESCRIPTION
**Pull request by chriscarter90**
Don't hardcode the admin username and password in the controller, and instead put them in ENV variables.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,9 +2,12 @@ source 'https://rubygems.org'
 
 ruby "2.2.2"
 
-gem 'rails', '4.2.2'
+gem 'dotenv-rails', :require => 'dotenv/rails-now'
+
 gem 'acts-as-taggable-on', '~> 3.4'
+gem "config", github: 'railsconfig/config'
```
Comment (charlieegan3) [inline Gemfile]: Could this be: `gem 'dotenv-rails', require: 'dotenv/rails-now'` ?

Comment (FriedSock) [inline Gemfile]: Why are you installing it from github?

Comment (chriscarter90) [inline Gemfile]: Good point, I stole these from another project.

Comment (chriscarter90) [inline Gemfile]: Fixed.

**Comment by otlaitil** [comment]
LGTM :+1: Happy to see this merged in.

**Pull request merged**